adapters/handlers/rest/db_users/handlers_db_users.go

```diff
@@ -156,6 +160,10 @@ func (h *dynUserHandler) getUser(params users.GetUserInfoParams, principal *mode
 		return users.NewGetUserInfoForbidden().WithPayload(cerrors.ErrPayloadFromSingleErr(err))
 	}
 
+	if !h.dbUserEnabled {
+		return users.NewGetUserInfoUnprocessableEntity().WithPayload(cerrors.ErrPayloadFromSingleErr(errors.New("db user management is not enabled")))
```

Comment:
this will return this newly defined 422 error, yes?

Reply:
yes!